kafka/metrics/measurable.py

```diff
@@ -0,0 +1,27 @@
+import abc
+
+
+class AbstractMeasurable(object):
+    """A measurable quantity that can be registered as a metric"""
+    @abc.abstractmethod
+    def measure(self, config, now):
+        """
+        Measure this quantity and return the result as a double
+
+        Arguments:
+            config (MetricConfig): The configuration for this metric
+            now (int): The POSIX time in milliseconds the measurement
```

Comment:
i think it's partially to make testing easier, but from what i've seen in the comments it's also a slight optimization. when you record or read a stat, you pass in the time and it's reused through the calls to check for expired stats or anything else that needs a timestamp, rather than calling to get the current (and slightly advanced) timestamp each time it is used.

could possibly be removed since we can mock time, but i don't have a good feel for how it's used in instrumentation yet.